catherine_feijoo/class2/bouncer.rb

```diff
@@ -0,0 +1,14 @@
+puts "how old are u?"
+age = gets.chomp.to_i
+
+def check_age(user_age)
+  if user_age<=20
+    puts "plz leave"
+  elsif user_age==21
+  	puts "ok i guess u can come in"
+  else
+  	puts "ofc u can come in"
+  end
+end
```

Comment:
Looks good! If you wanted to you could even shorten up this method a little bit more. Check it out:
```
def check_age(user_age)
if user_age =< 20
puts "please leave"
else
puts "okay I guess you can come in"
end
```